### PR TITLE
fix: Document a few more fields in trace context

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: general/src/protocol/contexts.rs [hunk 1]
```diff
@@ -670,6 +683,7 @@ fn test_trace_context_roundtrip() {
             );
             map
         },
+        ..Default::default()
```
Comment (jan-auer) [inline general/src/protocol/contexts.rs]: Please add all attributes to test the round trip. In the round trip tests we never use Default.

File: general/src/protocol/contexts.rs [hunk 2]
```diff
@@ -336,6 +342,13 @@ pub struct TraceContext {
     /// The ID of the span.
     pub span_id: Annotated<SpanId>,
 
+    /// The ID of the span enclosing this span.
+    pub parent_span_id: Annotated<SpanId>,
```
Comment (jan-auer) [inline general/src/protocol/contexts.rs:344]: Is it too late to name this shorter? It’s kinda strange that this has such a long name, but `op` is not typed out at the same time.

File: general/src/protocol/contexts.rs [hunk 3]
```diff
@@ -210,6 +210,12 @@ impl BrowserContext {
     }
 }
 
+/// Operation type such as `db.statement` for database queries or `http` for external HTTP calls.
+/// Tries to follow OpenCensus/OpenTracing's span types.
+///
+/// TODO typing ( union of predefined types and custom 'string' types)
```
Comment (jan-auer) [inline general/src/protocol/contexts.rs]: Probably fine if it’s not typed out, just like breadcrumbs. The only place where we would benefit from this is Sentry-types, probably.

**Review by jan-auer — APPROVED** [review]

File: general/src/protocol/contexts.rs (hunk 2 above)
Comment (untitaker) [reply general/src/protocol/contexts.rs:344]: That's true but it's quite engrained by now. I think we could rename it in semaphore if we needed to.

File: general/src/protocol/contexts.rs (hunk 3 above)
Comment (untitaker) [reply general/src/protocol/contexts.rs]: I'll remove the TODO. IMO the type alias is still useful for documentation

File: general/src/protocol/contexts.rs (hunk 1 above)
Comment (untitaker) [reply general/src/protocol/contexts.rs]: You're right, fixed

**Review by jan-auer — APPROVED** [review]

**Pull request merged**